projects/WTI-UI/src/app/modules/clarifications/components/new-clarification/new-clarification.component.ts

```diff
@@ -39,15 +39,19 @@ export class NewClarificationComponent implements OnInit, OnDestroy {
 
     this._teamService.postClarification(newClarification)
       .pipe(takeUntil(this._unsubscribe))
-      .subscribe(_ => {
-        this.newClarificationForm.reset();
-        this.close();
-        this._contestService.clarificationsUpdated.next();
-        this._uiHelper.alertOk('Clarification has been submitted successfully!');
-      }, (error: any) => {
-        console.error('error submitting new clarification');
-        console.error(error);
-      });
+      .subscribe({
+		next: value  => {
+	      this.newClarificationForm.reset();
+	      this.close();
+	      this._contestService.clarificationsUpdated.next();
+	      this._uiHelper.alertOk('Clarification has been submitted successfully! ');
+	    },
+	    error: (error: any) => {
+	      this._uiHelper.alertError(`Error submitting clarification! ${error}`);
+	    },
```

Comment:
I think it's good that you added the template string so it includes the error value in the alert popup... but I don't care for the fact that you eliminated the error being logged to the console.  The alert will go away if someone dismisses it.  If they then call a staff member over and say "I got an error; what's wrong?"  the staff member can no longer see the error (because it's no longer in the log).  I think errors should _always_ be logged.